tests/unit/prometheus_test.cc

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2024 Redpanda Data
+ */
```

Comment:
I wonder how upstream will view this?

Reply:
Let's see what they think when I upstream it.

Reply:
Added a license now with the fmt changes